nssp/delphi_nssp/run.py

```diff
@@ -72,18 +70,28 @@ def run_module(params):
         Nested dictionary of parameters.
     """
     start_time = time.time()
-    logger = get_structured_logger(
-        __name__,
-        filename=params["common"].get("log_filename"),
-        log_exceptions=params["common"].get("log_exceptions", True),
-    )
+    custom_run = params["common"].get("custom_run", False)
+    # logger doesn't exist yet means run_module is called from normal indicator run (instead of patching)
+    if not logger:
+        logger = get_structured_logger(
+            __name__,
+            filename=params["common"].get("log_filename"),
+            log_exceptions=params["common"].get("log_exceptions", True),
+        )
+        if custom_run:
```

Comment:
Are you explicitly using the custom_run as patch vs regular run?

Reply:
As of now, yes.

Reply:
Can you make a comment about it? Something like custom flag only supported for patching, then it makes more sense for the next comment that I had before.

Reply:
Is there a reason? Like wanting to make things more simple/ unnecessary extra work for non patch custom runs that wouldn't be used? A small comment of the reason would be nice. Like if this was because of a technical limitation or non essential/necessary

Reply:
no reason. When I read your suggestion to add the custom_run flag, I just thought of the patch use case for it, so I wrote the flag as that. I would imagine if we do other kinds of custom run in the future, we can change it then.

Reply:
Since it seems like people use custom start and end date can we change the logic to more align with the behavior that google symptom has (custom flag is on, but on regular run) 

For custom regular run, we could also log the start and end date and like a messaging saying custom run